docs/spock_functions/functions/spock_max_proto_version.md

```
@@ -0,0 +1,45 @@
+The protocol version is returned as an integer value. Higher numbers
+indicate newer protocol versions with additional features.
+
+This is a read-only query function that does not modify data.
+
+### ARGUMENTS
+
+This function takes no arguments.
+
+### EXAMPLE
+
+postgres=# SELECT spock.spock_min_proto_version();
```

markdownlint-cli2 (0.20.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)